src/config.c

```diff
@@ -2601,6 +2601,61 @@ int allowProtectedAction(int config, client *c) {
            (config == PROTECTED_ACTION_ALLOWED_LOCAL && islocalClient(c));
 }
 
+
+static int setConfigLatencyPercentileOption(typeData data, sds *argv, int argc, const char **err) {
+    UNUSED(data);
+
+    /* Special case: treat single arg "" as zero args indicating empty percentile configuration */
+    if (argc == 1 && !strcasecmp(argv[0],""))
+        argc = 0;
+
+    resetServerLatencyPercentileParams();
```

Comment:
i still think i'd rather eliminate `resetServerLatencyPercentileParams` and `appendServerLatencyPercentileParams`, and just embed their code here.
i think it'll make the code clearer.
I suppose the reason it's here is because it was copied from "save" which had other concerns in the past.

Reply:
addressed in the latest commit oran